### ISSUE TITLE
Can't open terminal into current directory.

### ISSUE THREAD

**Issue by jaychaturvedi** [platformio/platformio-atom-ide-terminal#633]
Everytime the terminal open in  directory "C:\WINDOWS\System32\WindowsPowerShell\v1.0" instead of the current working directory.

**Comment by the-j0k3r** [comment]
You can, it works right click dir and open with atom. Then when you open the terminal panel its in that dir.

Also open folder in Atom does similar, again then open the terminal panel.

**Comment by MrAmbiG** [comment]
When you open atom it usually has previous projects and folder open in it. So it wont work with preloaded folders/projects. So In order for me to open terminal in that place, i have to remove all projects/folders that are there in the atom already and re add them?! It does not seem like a productive way of doing it.

**Comment by the-j0k3r** [comment]
@MrAmbiG I agree in part, but  cant do anything myself at this time, though if you can, feel free to design and implement a menu entry solution and submit a pull request. =).

**Comment by MrAmbiG** [comment]
I appreciate the reply. It being the most used plugin in this category, i
am pretty sure the author has a lot of requests in pipeline. I would if i
could but i guess i will wait.

Thanks and Regards,
https://ambig.one/2/
*Gajendra D Ambi *|AWS-CSAA, VCAP6-DCA, 2xVCA, 7xVCP, VCE-CIA, EMC-ISA,
Comptia Server+/A+, MCSE, 2xMCTS vExpert (2017/2018) |

**Comment by the-j0k3r** [comment]
Read #543 which means the author is out. Its up to the community now.
